lib/coll.gd

```diff
@@ -2107,7 +2107,7 @@ DeclareOperation( "ListOp", [ IsListOrCollection, IsFunction ] );
 ##  <C>Length( <A>new</A> )</C> is the number of elements of
 ##  <A>listorcoll</A>,
 ##  and <A>new</A> contains the elements in sorted order,
-##  w.r.t.&nbsp;<C>&lt;=</C>.
+##  w.r.t.&nbsp;<C>&lt;=</C> or <A>func</A> if it is specified.
```

Comment:
I realize this was here before, but: why does this say `<=` and not `<` as in the documentation for `Sort` etc? That may matter in case the list contains two copies of the same element... is `func` then supposed to return true or false? Hmm, seeing that this actually calls `Sort`, it seems the current documentation is actually incorrect!

Overall, this text doesn't really make it clear what `func` is about. The quickest way to make this clearer might be to reference the documentation for `Sort` here; alternatively, one could also copy text from there, but seeing as the implementation just calls `Sort`, I think it would make a lot of sense to just link to the Sort documentation, this way, they'll always be automatically in sync.

Perhaps like this:
```suggestion
##  w.r.t.&nbsp;<C>&lt;</C> or <A>func</A> if it is specified.
##  For details, please refer to <Ref Oper="Sort"/>.
```